java/java.lsp.server/nbcode/branding/modules/org-openide-loaders.jar/org/netbeans/modules/openide/loaders/Bundle.properties

```diff
@@ -0,0 +1,21 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# yes/no/ask
+ASK_OnSaving=no
+# yes/no/ask
+ASK_OnClosing=no
```

Comment:
Q: shouldn't be there some dependency on loaders >= 1.80 ?

Reply:
Right. I did check `java/java.lsp.server/nbcode/integration/nbproject/project.xml` and there is no dependency on `org.openide.loaders`. I can add one.

Reply:
Right. I did check `java/java.lsp.server/nbcode/integration/nbproject/project.xml` and there is no dependency on `org.openide.loaders`. I can add one.

Reply:
Fixed in cc1eb58196680